src/timePicker/TimePicker.tsx

```diff
@@ -105,6 +96,12 @@ export default class TimePicker extends PureComponent<TimePickerProps, TimePicke
     this.props.onChange({ name, value })
   }
 
+  componentDidMount() {
```

Comment:
Do we have a specific style when focussed?

Reply:
I'm adding it (I'll update the description)